app/helpers/report_helper.rb

```diff
@@ -3,15 +3,24 @@
 #   the COPYRIGHT file.
 
 module ReportHelper
+  def get_reported_guid(id, type)
```

Comment:
Leave off `get_`, `set_` (replace with `=` suffix) and `is_` (replace with `?` suffix) prefixes in Ruby. Also this should be fine with being a method on the Report model, becoming `report.reported_author.guid` in the view.